app/lib/common/widgets/chat/convo_hierarchy_card.dart

```diff
@@ -89,14 +97,17 @@ class ConvoHierarchyCard extends ConsumerWidget {
     this.contentPadding = const EdgeInsets.all(15),
     this.shape,
     this.withBorder = true,
+    this.showIconIfSuggested = false,
+    this.trailing,
   });
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final roomId = roomInfo.roomIdStr();
-    final avatarInfo = ref.watch(spaceHierarchyAvatarInfoProvider(roomInfo));
+    final avatarInfo = ref.watch(roomHierarchyAvatarInfoProvider(roomInfo));
     final topic = roomInfo.topic();
     final subtitle = topic?.isNotEmpty == true ? Text(topic!) : null;
+    bool showSuggested = showIconIfSuggested && roomInfo.suggested();
```

Comment:
because that would mean the outer would have to check whether the room is suggested, rather than just configuring whether it should be shown _if_ the room was suggested. I opted for just having the outer decide this is a place to show the icon at all or never and then have the code here figure out whether it is to be shown.